Project.toml

```
@@ -0,0 +1,30 @@
+name = "ReferenceTests"
+uuid = "324d217c-45ce-50fc-942e-d289b448e8cf"
+authors = ["Evizero <stocker.christof@gmail.com>"]
```

Given all the changes he made I'd like to add @oxinabox to the list of authors

```suggestion
authors = ["Christof Stocker <stocker.christof@gmail.com>", "Lyndon White <oxinabox@ucc.asn.au>"]
```